src/main/flight/position.c

```diff
@@ -208,7 +208,7 @@ void calculateEstimatedAltitude(void)
     DEBUG_SET(DEBUG_ALTITUDE, 3, estimatedVario);
 #endif
     DEBUG_SET(DEBUG_RTH, 1, lrintf(displayAltitudeCm / 10.0f));
-    DEBUG_SET(DEBUG_BARO, 3, lrintf(baroAltCm / 10.0f));
+    DEBUG_SET(DEBUG_BARO, 3, lrintf(baroAltCm));
```

Comment:
Better to just delete this line altogether.
Baro altitude is already written to baro debug 3 in barometer.c:
`DEBUG_SET(DEBUG_BARO, 3, lrintf(baro.altitude));            // cm`